http.go

```diff
@@ -17,11 +17,12 @@ type resultContainer struct {
 	Success  bool      `json:"success"`
 	Message  string    `json:"message"`
 	Comments []Comment `json:"comments,omitempty"`
+	Count    *int       `json:"count,omitempty"`
```

Comment:
Actually I did it like you suggest originally, but then changed to this since I wanted the count to be left out for a POST request, and this was a way to do so, since the count could also be nil, but having given it some thought, it makes sense to just return the updated count after a successful POST. So I will update the PR to incorporate that & your other suggestions. Thanks for the review!